tests/core/pyspec/eth2spec/test/helpers/attestations.py

```diff
@@ -105,13 +105,24 @@ def get_valid_attestation(spec,
     if index is None:
         index = 0
 
-    attestation_data = build_attestation_data(spec, state, slot=slot, index=index, beacon_block_root=beacon_block_root)
+    if isinstance(index, (list, tuple)):
```

Comment:
I think it would be cleaner to have a separate helper that capable of creating multi committee attestations and reuses `get_valid_attestation` as the underlying call